NEWS.md

```diff
@@ -32,6 +36,14 @@ the derivation is called for all subsets. (#2645)
     
     **Phase 4 (removed)**
 
+- The values of the variable specified for `tmp_obs_nr_var` in
```

Comment:
should we move this up in the breaking changes, i.e. above the deprecation stuff?

Reply:
Yes, good idea.